src/templates/hi/2019/base.html

```diff
@@ -0,0 +1,188 @@
+
+{% block help_translate %}अनुवाद में मदद करें{% endblock %}
+{% block translation_not_available %}अंग्रेजी - अंग्रेजी में उपलब्ध नहीं है{% endblock %}
+{% block language_switcher %}भाषा परिवर्तन करें{% endblock %}
+{% block year_switcher %}वर्ष परिवर्तन करें{% endblock %}
+
+{% block home %}होम{% endblock %}
+{% block table_of_contents_title %}विषय सूची{% endblock %}
+{% block contributors_title %}योगदानकर्ता{% endblock %}
+{% block methodology_title %}कार्यप्रणाली{% endblock %}
+
+{% block part %}भाग{% endblock %}
+{% block chapter %}अध्याय{% endblock %}
+
+{% block start_exploring %}खोज शुरू करें{% endblock %}
+
+{% block menu_title %}पृष्ठ सूची{% endblock %}
+{% block open_the_menu %}मेनू को खोलें{% endblock %}
+{% block close_the_menu %}मेनू को बंद करें{% endblock %}
+
+{% block index_nav_title %}अध्याय विषय सूची{% endblock %}
+{% block index_title %}निर्देशिका{% endblock %}
+{% block open_the_index %}विषय सूची को खोलें{% endblock %}
+{% block close_the_index %}विषय सूची को बंद करें{% endblock %}
+
+{% block and %}और{% endblock %}
+{% block comma %}, {% endblock %}
+
+{% block open %}खोलें{% endblock %}
+{% block close %}बंद करें{% endblock %}
+
+{% macro figure_text(metadata, id) %}खंड {{ figure_id(metadata, id) }}.{% endmacro %}
+{% macro show_description(metadata, id) %}खंड का विवरण देखें {{ figure_id(metadata, id) }}{% endmacro %}
+{% macro hide_description(metadata, id) %}खंड का विवरण छिपाएँ {{ figure_id(metadata, id) }}{% endmacro %}
+
+{% block written_by_before %}द्वारा लिखित{% endblock %}
+{% block reviewed_by_before %}द्वारा समीक्षित{% endblock %}
+{% block translated_by_before %}द्वारा अनुवादिद्त{% endblock %}
+{% block analysis_by_before %}द्वारा विश्लेषित{% endblock %}
+{% block written_by_after %}{% endblock %}
+{% block reviewed_by_after %}{% endblock %}
+{% block translated_by_after %}{% endblock %}
+{% block analysis_by_after %}{% endblock %}
+
+{% block author %}लेखक{% endblock %}
+{% block authors %}लेखकों{% endblock %}
+
+{% macro onTwitter(twitterHandle) %}@{{twitterHandle}} Twitter पर{% endmacro %}
+{% macro onGitHub(gitHubHandle) %}{{gitHubHandle}} GitHub पर{% endmacro %}
+{% macro onLinkedIn(authorName) %}{{authorName}} LinkedIn पर{% endmacro %}
+{% macro website(authorName) %}{{authorName}} वेबसाइट{% endmacro %}
+
+{% macro edition(year) %}{{ year }} संस्करण{% endmacro %}
+
+{% block footer_title %}फुटर नेविगेशन{% endblock %}
+{% block copyright %}© Web Almanac. <a href="https://github.com/HTTPArchive/almanac.httparchive.org/blob/main/LICENSE">Apache 2.0</a> के तहत लाइसेंस प्राप्त है।{% endblock %}
+{% block accessibility_statement %}अभिगम्यता विवरण{% endblock %}
+
+{% block featured_chapter %}विशेष अध्याय{% endblock %}
+
+{# Check if read_chapter already defined in child template as macros can't be overridden #}
+{% if not read_chapter %}
+{% macro read_chapter(chapter) %}अध्याय <span class="featured-chapter-name">{{ chapter }}</span> पढ़ें{% endmacro %}
+{% endif %}
```

Comment:
New changes we're about to add in #1452 

I used Google Translate so can you check these?

```suggestion
{% block featured_chapter %}विशेष अध्याय{% endblock %}
{% block featured_chapter_last_year %}विशेष अध्याय<br />2019 Web Almanac से{% endblock %}

{# Check if read_chapter already defined in child template as macros can't be overridden #}
{% if not read_chapter %}
{% macro read_chapter(chapter) %}अध्याय <span class="featured-chapter-name">{{ chapter }}</span> पढ़ें{% endmacro %}
{% endif %}
{% if not read_last_years_chapter %}
{% macro read_last_years_chapter(chapter) %}{{ year|int -1}} अध्याय <span class="featured-chapter-name">{{ chapter }}</span> पढ़ें{% endmacro %}
{% endif %}
```

Reply:
Will do that in my next commit :)